modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/select_parent.jsp

```diff
@@ -58,7 +58,7 @@ kbSelectParentDisplayContext.populatePortletBreadcrumbEntries(currentURLObj);
 							"title", parentTitle
 						).build()
 					%>'
-					value='<%= (kbSelectParentDisplayContext.getParentResourceClassNameId() == kbFolderClassNameId) ? "choose-this-folder" : "choose-this-article" %>'
+					value='<%= (kbSelectParentDisplayContext.getParentResourceClassNameId() == kbFolderClassNameId) ? "select-this-folder" : "select-this-article" %>'
```

Comment:
FYI bookmarks and recycle bin still use "Choose This Folder." I am not sure if those needed to be updated (if so they will break tests)

Reply:
I don't think those should be updated for now...